decidim-admin/app/forms/decidim/admin/participatory_process_form.rb

```diff
@@ -25,7 +25,7 @@ class ParticipatoryProcessForm < Form
       attribute :slug, String
       attribute :hashtag, String
       attribute :promoted, Boolean
-      attribute :scope_id, Integer
+      attribute :scope_id, String
```

Comment:
Yes, at first I was using the global scope as an option, so I did what you said. But, finally I changed it to use global scope as the empty selection, which I think is more elegant. But I forgot to rollback the code change that you are commenting. I've reviewed it and I think now it is consistent.